sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseDataForm.java

```diff
@@ -169,20 +178,29 @@
 					fluidRowLocs(CaseDataDto.CLINICIAN_PHONE, CaseDataDto.CLINICIAN_EMAIL) +
 					loc(PAPER_FORM_DATES_LOC) +
 					fluidRowLocs(CaseDataDto.DISTRICT_LEVEL_DATE, CaseDataDto.REGION_LEVEL_DATE,
-							CaseDataDto.NATIONAL_LEVEL_DATE)
-					+ loc(GENERAL_COMMENT_LOC) + fluidRowLocs(CaseDataDto.ADDITIONAL_DETAILS);
+							CaseDataDto.NATIONAL_LEVEL_DATE);
+
+	private static final String FOLLOWUP_LAYOUT =
+			fluidRowLocs(CaseDataDto.FOLLOW_UP_STATUS, CANCEL_OR_RESUME_FOLLOW_UP_BTN_LOC, LOST_FOLLOW_UP_BTN_LOC) +
+					fluidRowLocs(4, CaseDataDto.FOLLOW_UP_UNTIL, 8, CaseDataDto.OVERWRITE_FOLLOW_UP_UNTIL) +
+					fluidRowLocs(CaseDataDto.FOLLOW_UP_COMMENT);
+
+	private static final String TAIL_HTML_LAYOUT =
```

Comment:
Kinda weird naming, maybe rename to COMMENTS_HTML_LAYOUT

Reply:
OK